tests/recipient_validation/test_phone_number.py

```diff
@@ -473,15 +464,15 @@ def test_get_normalised_format_works_for_international_numbers(self, phone_numbe
     @pytest.mark.parametrize(
         "phone_number, expected_formatted",
         [
-            ("07900900123", "07900 900123"),  # UK
```

Comment:
tv numbers not being valid would be a bother - we'd have to have a huddle about this on tech catch up sometime - but if I understand correctly, for now the new validation will only be used for OneLogin, so not urgent.

Reply:
We mentioned this in a Trello card for introducing new validation code more widely: 

https://trello.com/c/nLANAG47/62-replace-old-validation-code-entirely-with-new-code-and-remove-old-validation-code-from-utils